### DISCUSSION TITLE
Error occurred when executing KSampler:Expected query, key, and value to have the same dtype, but got query.dtype: struct c10::Half key.dtype: float and value.dtype: float instead.

### DISCUSSION THREAD

**Discussion by lilijia1982** [cubiq/ComfyUI_IPAdapter_plus#618] [General]
在运行时候报的错误，完全看不懂，求助各位大神。
[
![微信图片_20240702133320](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/173114868/0c35373a-79b1-4fcf-bab9-20bc0f64c146)
](url)

**Comment by cubiq** [comment]
run comfy in --force-fp16 mode
